doc/Setup.md

```diff
@@ -0,0 +1,35 @@
+# How to Run the OAuth Agent Locally
+
+Follow the below steps to get set up for developing and testing the OAuth Agent itself.
+
+## Prerequisites
+
+Ensure that these tools are installed locally:
+
+- [Node.js](https://nodejs.org/en/download/)
+- [jq](https://stedolan.github.io/jq/download/)
+
+## Build and Run the OAuth Agent
+
+Run these commands from the root folder and the API will then listen on HTTP over port 3000:
+
+```bash
+npm install
+npm dev
+```
+
+Test that the API is contactable by running this command from the root folder:
```

Comment:
requires to have `api.example.local` set up. Add a note on that or change to `localhost`

Reply:
Yes, I meant to change it to localhost. Must have forgotten...